code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -184,7 +179,7 @@ public void updateAccount(Account account, String name, String kind, String exte
             projects.addAll(create()
                     .selectFrom(ACCOUNT)
                     .where(ACCOUNT.KIND.eq(ProjectConstants.TYPE)
-                            .and(ACCOUNT.STATE.eq(CommonStatesConstants.ACTIVE)))
+                            .and(ACCOUNT.STATE.ne(CommonStatesConstants.PURGED)))
```

Comment:
Untill we do the refactor we can leave this as is. Since we are removing purged and purging.